src/test/java/com/axlabs/neo/grantshares/GovernanceMembersTest.java

```diff
@@ -26,32 +27,14 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
-import static com.axlabs.neo.grantshares.util.TestHelper.ADD_MEMBER;
-import static com.axlabs.neo.grantshares.util.TestHelper.ALICE;
-import static com.axlabs.neo.grantshares.util.TestHelper.BOB;
-import static com.axlabs.neo.grantshares.util.TestHelper.CALC_MEMBER_MULTI_SIG_ACC;
-import static com.axlabs.neo.grantshares.util.TestHelper.CHARLIE;
-import static com.axlabs.neo.grantshares.util.TestHelper.DENISE;
-import static com.axlabs.neo.grantshares.util.TestHelper.GET_MEMBERS;
-import static com.axlabs.neo.grantshares.util.TestHelper.MEMBER_ADDED;
-import static com.axlabs.neo.grantshares.util.TestHelper.MEMBER_REMOVED;
-import static com.axlabs.neo.grantshares.util.TestHelper.PHASE_LENGTH;
-import static com.axlabs.neo.grantshares.util.TestHelper.PROPOSAL_EXECUTED;
-import static com.axlabs.neo.grantshares.util.TestHelper.REMOVE_MEMBER;
-import static com.axlabs.neo.grantshares.util.TestHelper.assertAborted;
-import static com.axlabs.neo.grantshares.util.TestHelper.createAndEndorseProposal;
-import static com.axlabs.neo.grantshares.util.TestHelper.createMultiSigAccount;
-import static com.axlabs.neo.grantshares.util.TestHelper.prepareDeployParameter;
-import static com.axlabs.neo.grantshares.util.TestHelper.voteForProposal;
+import static com.axlabs.neo.grantshares.util.TestHelper.*;
```

Comment:
I prefer if we wouldn't use wildcard imports. While it increases the line number in some files, it improves code readability as it is clear which specific import is used for what, and maintains consistency across our projects.

Suggestion: Since I'm already talking about code-style. I've also noticed some inconsistent line breaks in the PR (e.g., functions in GrantSharesGov are wrapped if they exceed 120 chars, but lines in this file are not). Maybe it'd make sense to commit a code-style file in the project, so that we can define a format once and don't need to worry about it further along.

Reply:
is there a codestyle repo for AxLabs? or are you using google's codestyle - https://github.com/google/styleguide ?

Reply:
We have one in the `neow3j` repo [here](https://github.com/neow3j/neow3j/tree/main/.idea/codeStyles).

Reply:
We should consider removing `<option name="IF_BRACE_FORCE" value="3" />` or set the value to `0` because validating function parameters and just aborting happens quite frequently and imho it is fine to have this in a single line since, otherwise, it bloats each function unnecessarily if there are several validity checks in the beginning of a function.

Reply:
It's weird. I've copied the codeStyles folder from neow3j and it doesn't apply the wrap at 120 characters. So i haven't pushed b/c I want that to be in the codestyle file as well

On the other hand, with respect to this particular set of imports and the way they are used, I'd thought about separating the static fields in static classes inside TestHelper, for clarity, in that some are methods, events, members, parameter names, etc.